.eslintrc.cjs

```diff
@@ -229,6 +229,9 @@ module.exports = {
           config: path.join(__dirname, './packages/web/docs/tailwind.config.ts'),
         },
       },
+      rules: {
+        'import/extensions': 'off',
+      },
```

Comment:
@kamilkisiela gave me a leave to disable ESLint rules that disturb my work, and this one has **only false positives** in my experience. TypeScript checks this. Next.js checks this (by the virtue of building stuff and resolving modules, right?)